src/app/workflow/refresh-workflow-organization/refresh-workflow-organization.component.ts

```diff
@@ -54,6 +58,24 @@ export class RefreshWorkflowOrganizationComponent extends RefreshOrganizationCom
     this.isRefreshing$ = this.alertQuery.showInfo$;
   }
 
+  openConfirmationDialog() {
+    const confirmationDialogData: ConfirmationDialogData = {
+      title: 'Refresh Organization',
+      message: `Are you sure you wish to refresh the organization?
+                It has no effect on GitHub Apps workflows.`,
```

Comment:
"GitHub Apps workflows" -> I think we use it as a shorthand, but I'm not sure it will click right away for end-users.

Perhaps, "It has no effect on workflows already being synchronized via the Dockstore GitHub App"?